HW5Class6/Task3.py

```diff
@@ -0,0 +1,10 @@
+first_num = int(0)
+last_num = int(101)
```

Comment:
you don't need to use `int` here, 0 and 101 are already integers
```suggestion
first_num = 0
last_num = 101
```